presto-main/src/test/java/com/facebook/presto/server/remotetask/TestHttpRemoteTask.java

```diff
@@ -245,12 +246,14 @@ public void testHTTPRemoteBadTaskSize()
         httpRemoteTaskFactory.stop();
 
         assertTrue(remoteTask.getTaskStatus().getState().isDone(), format("TaskStatus is not in a done state: %s", remoteTask.getTaskStatus()));
-        assertEquals(getOnlyElement(remoteTask.getTaskStatus().getFailures()).getMessage(), "TaskUpdate size of 1.97kB has exceeded the limit of 1kB");
+        assertThat(getOnlyElement(remoteTask.getTaskStatus().getFailures()).getMessage())
+                .matches("TaskUpdate size of .+? has exceeded the limit of 1kB");
```

Comment:
do we expect this change to impact the taskUpdate size? by how much (might it cause problems)?

Reply:
The new field increases the task update size by a couple of bytes so the test fails.

I was trying to understand the intention of this test, and it seems like it was added to check if the error message is properly formatted. 

There's a test down below that injects a JSON of a certain size and checks if the size matches exactly. It felt like checking here for the exact size may not be necessary.

@misterjpapa could you please take a look and advice?